### ISSUE TITLE
Circuit Design

### ISSUE THREAD

**Issue by mgood13** [mgood13/mgood13_Design_Skills_Project#4]
Full Disclosure: I made a circuit that was totally functional and would work for this project but that wasn't what I really wanted to make for the project. Now I've made the real circuit and want to use that instead. So 2 questions:
1. Could I upload that circuit now and get some feedback on it?
2. Do you know what kind of Decade Counter IC is in the lab so that I can find it on Eagle? (This one isn't that important as it will show up on the library when that is finished, but just figured I would ask)

**Comment by mlp6** [comment]
Yep... I strongly encourage going after the expanded scope version of the circuit!!

Lets get Matt B. added as a  Collaborator on this project to see if he can comment on the counter... I will also see him this morning and can ask.

**Comment by mlp6** [comment]
@sphaerobolus can you comment?
